sf-docker/.env

```diff
@@ -25,9 +25,9 @@ NoSqlDataStorageDisabled=false
 
 # Using template for Gstreamer pipeline
 #jetson
-GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
+#GstPipelineTemplate=uridecodebin uri={0} source::latency=0 ! queue max-size-buffers=1 leaky=downstream ! nvvidconv ! video/x-raw, format=(string)BGRx ! videoconvert ! video/x-raw, format=(string)BGR ! appsink
```

Comment:
This means that the jetson `run-jetson.sh` script won't work out-of-the-box. We could consider splitting this configuration to two separate variables e.g. `GstPipelineTemplate_linux` and `GstPipelineTemplate_jetson` and then using them like this:

in `docker-compose.yml`
```
- GstPipelineTemplate=${GstPipelineTemplate_linux}
```

in `jetson-docker-compose.yml`
```
- GstPipelineTemplate=${GstPipelineTemplate_jetson}
```